Products/ZenModel/ZenPackLoader.py

```diff
@@ -458,24 +458,29 @@ def load(self, pack, app):
                         raise Exception('Could not find unique name for trigger: "%s".' % trigger_conf['name'])
 
                     log.info('Creating trigger: %s' % test_name)
-                    tf.createTrigger(test_name, uuid=trigger_conf['uuid'], rule=trigger_conf['rule'])
 
+                    try:
+                        tf.createTrigger(test_name, uuid=trigger_conf['uuid'].lower(), rule=trigger_conf['rule'])
+                    except ServiceResponseError as e:
+                        log.warning("Could not create trigger '%s': %s" % (test_name, e))
 
             for notification_conf in notifications:
-                
+
                 existing_notification = guidManager.getObject(str(notification_conf['guid']))
 
                 if existing_notification:
                     log.info("Existing notification found, updating: %s" % existing_notification.id)
-                    
+
                     subscriptions = set(existing_notification.subscriptions + notification_conf['subscriptions'])
                     notification_conf['uid'] = '/zport/dmd/NotificationSubscriptions/%s' % existing_notification.id
-                    notification_conf['subscriptions'] = list(subscriptions)
+                    # ensure uuid's are lowercase
+                    notification_conf['subscriptions'] = [x.lower() for x in list(subscriptions)]
                     notification_conf['name'] = existing_notification.id
-                    tf.updateNotification(**notification_conf)
+                    try:
+                        tf.updateNotification(**notification_conf)
+                    except ServiceResponseError as e:
+                        log.warning("Could not update notification '%s': %s" % (existing_notification.id, e))
```

Comment:
You are changing the behavior on error here. Do you not wan't the install to end?